### PR TITLE
My memes screen

### PR DESCRIPTION
**Pull request by Ozzorany**

### CODE REVIEW

**Comment by Ozzorany** [comment]
@yarivg  The whole task of my memes screen is not finished yet- but after this pr, you can start and use the meme service & try to get the images from the storage of firebase

File: app/src/main/java/com/example/extrememe/MyMemesFragment.java [hunk 1]
```diff
@@ -12,35 +12,44 @@
...
+            public void onCallback(List<Meme> memes) {
+                data = memes;
+                adapter.data = data;
+                adapter.notifyDataSetChanged();
+
+
+                adapter.setOnClickListener(new MyMemesAdapter.OnItemClickListener() {
+                    @Override
+                    public void onItemClick(int position) {
+                        Log.d("TAG","meme " + position + " was clicked ");
```
Comment (yarivg) [inline app/src/main/java/com/example/extrememe/MyMemesFragment.java]: why tag?

File: app/src/main/java/com/example/extrememe/services/DatabaseDataLoader.java [hunk 2]
```diff
@@ -17,4 +20,12 @@ public static FirebaseFirestore getDB() {
 
         return fireStoreDB;
     }
+
+    public static StorageReference getStorageRef(String url) {
+        if(storageRef == null) {
+            storageRef = storage.getReferenceFromUrl(url);
```
Comment (yarivg) [inline app/src/main/java/com/example/extrememe/services/DatabaseDataLoader.java]: the storage object might be null - android is known for static object get deliminated during lifecycle
https://stackoverflow.com/questions/9541688/static-variable-null-when-returning-to-the-app

- [x]  FirebaseStorage.getInstance().getReferenceFromUrl(url)

**Review by yarivg — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/example/extrememe/MyMemesFragment.java (hunk 1 above)
Comment (Ozzorany) [reply app/src/main/java/com/example/extrememe/MyMemesFragment.java]: Just wanted to check if the listener works well.
I've just removed it.

**Pull request merged**